src/lib/input/autosize.ts

```diff
@@ -19,33 +19,28 @@ export class MdTextareaAutosize implements AfterViewInit {
   private _minRows: number;
   private _maxRows: number;
 
-  /** @deprecated Use mdAutosizeMinRows */
-  @Input()
+  @Input('mdAutosizeMinRows')
   get minRows() { return this._minRows; }
 
   set minRows(value: number) {
     this._minRows = value;
     this._setMinHeight();
   }
 
-  /** @deprecated Use mdAutosizeMaxRows */
-  @Input()
+  @Input('mdAutosizeMaxRows')
   get maxRows() { return this._maxRows; }
-
   set maxRows(value: number) {
     this._maxRows = value;
     this._setMaxHeight();
   }
 
-  /** Minimum number of rows for this textarea. */
-  @Input()
-  get mdAutosizeMinRows(): number { return this.minRows; }
-  set mdAutosizeMinRows(value: number) { this.minRows = value; }
+  @Input('matAutosizeMaxRows')
```

Comment:
This is now private but relied on in internal tests. Do you want to make it deprecated for use and then make private after that?